scripts/configs/template.conf

```diff
@@ -33,10 +35,13 @@ SampleNDRange = # currently unsupported
 
 [Harness]
 Name = harness_stencil
-Args = -p 2 -s 1024 
+Args = -s 1024
+#partially moved to environment.conf
```

Comment:
Ok I understand the idea but please move the OpenCL config back here because although it is coupled to the specific system, the exploration.conf should contain all parameters that influence the times measured and that definitely includes the OpenCL Device and Platform (but for example not really the path to lift, thats why we moved it out to env.conf)